### ISSUE TITLE
upgraded account but no items

### ISSUE THREAD

**Issue by stephbjen** [dalaranwow/dalaran-wow#4045]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
upgraded to premium account about 2 weeks ago boosted to 70 and was suppose to get 2 heirloom items and tomb of cold weather flying. Have not received heirloom items of tomb of cold weather.
**Expected behaviour**: Tell us what should happen instead
please give me 2 heirloom items (plate shoulders and chest) as well as tomb of cold weather flying
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issuetracker is for ingame/servside issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.
